pages/installation/os/centos.rst

```diff
@@ -111,7 +111,8 @@ If you're using SELinux on your system, you need to take care of the following s
     - Elasticsearch (only if the HTTP API is being used): ``sudo semanage port -a -t http_port_t -p tcp 9200``
 - Allow using MongoDB's default port (27017/tcp): ``sudo semanage port -a -t mongod_port_t -p tcp 27017``
 
-If you run a single server environment with :ref:`NGINX or Apache proxy <configuring_webif_nginx>`, enabling the Graylog REST API is enough. All other rules are only required in a multi-node setup.
+If you run a single server environment with :ref:`NGINX or Apache proxy <configuring_webif_nginx>`, enabling the Graylog REST API is enough. All other rules are only required in a multi-node setup. 
+Having SELinux disabled during installation and enable it later, forces you to check and controll all policy that need to be present for MongoDB, Elasticsearch and Graylog.
```

Comment:
I would rewrite the sentence:
"Having SELinux disabled during installation and enabling it later, requires you to manually check the policies for MongoDB, Elasticsearch and Graylog."